examples/rhoai-quickstart/feast-demo-quickstart.ipynb

```diff
@@ -0,0 +1,1530 @@
+     "output_type": "stream",
+     "text": [
+      "total 36K\n",
+      "drwxr-xr-x. 4 1001130000 1001130000 4.0K Sep 24 03:07 .\n",
+      "drwxr-xr-x. 3 1001130000 1001130000 4.0K Sep 24 03:07 ..\n",
+      "drwxr-xr-x. 2 1001130000 root       4.0K Sep 24 03:07 data\n",
+      "-rw-r--r--. 1 1001130000 1001130000 5.2K Sep 24 03:07 example_repo.py\n",
+      "-rw-r--r--. 1 1001130000 1001130000  372 Sep 24 03:07 feature_store.yaml\n",
+      "-rw-r--r--. 1 1001130000 1001130000    0 Sep 23 16:09 __init__.py\n",
+      "drwxr-xr-x. 2 1001130000 1001130000 4.0K Sep 23 16:09 __pycache__\n",
+      "-rw-r--r--. 1 1001130000 1001130000 4.3K Sep 23 16:09 test_workflow.py\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Inspect the feast repo path files. Going to describe each file/folder purpose.\n",
+    "!ls -lah"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2fa8d4c3-8f7d-4758-8048-945ebd0af53f",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Now the feast repo has been created for you. Inspect the below files.\n",
+    "\n",
+    "`data` contains the parquet file data used to demonstrate this example.\n",
+    "\n",
+    "`example_repo.py` file will have the code to create feast objects such as FeatureView, FeatureServices and OnDemandFeatureViews required to demonstrate this example.\n",
+    "[my_feast_project/feature_repo/example_repo.py](./my_feast_project/feature_repo/example_repo.py)\n",
+    "\n",
+    "`feature_store.yaml` file will have all the configurations related to feast.\n",
+    "[my_feast_project/feature_repo/feature_store.yaml](./my_feast_project/feature_repo/feature_store.yaml)\n",
+    "\n",
+    "`test_workflow.py` contains the python code to demonstrate run all key Feast commands, including defining, retrieving, and pushing features.\n",
+    "[my_feast_project/feature_repo/test_workflow.py](./my_feast_project/feature_repo/test_workflow.py)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "f0a63ea0-93fa-4d4e-b12d-6caef86478f9",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "project: my_feast_project\n",
+       "# By default, the registry is a file (but can be turned into a more scalable SQL-backed registry)\n",
+       "registry: data/registry.db\n",
+       "# The provider primarily specifies default offline / online stores & storing the registry in a given cloud\n",
+       "provider: local\n",
+       "online_store:\n",
+       "    type: sqlite\n",
+       "    path: data/online_store.db\n",
+       "entity_key_serialization_version: 2\n"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "more feature_store.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81c5bc3c-8e50-410a-8a16-f92b1f4f4d08",
+   "metadata": {},
+   "source": [
+    "The default feature store configuration uses the sqllite as registry and online store."
```

Comment:
Actually, the `local` provider also creates an offline store that is based on Dask, though it's documented as `File` here:
https://docs.feast.dev/reference/providers/local.

You can prove this with few lines of python:
```py
from feast.feature_store import FeatureStore
store = FeatureStore(".")
print(store.config.offline_store)
```

Reply:
I have added more details to the description. Please let me know if it is not very clear.